src/main/java/frc/robot/commands/utilities/TimeRotation.java

```diff
@@ -10,6 +10,7 @@
   private double m_power;
   private final Timer m_timer = new Timer();
 
+  // TODO: Rewrite this command to be more accurate
```

Comment:
What was inaccurate about it?

Reply:
We don't account for the acceleration in the beginning which might throw off the calculated kV value? I'm not entire sure though.

Reply:
We're talking about TimeRotation(), which uses ArcadeDrive internally...it's not doing Profiled path following / feed forward.